src/patternfly/components/Button/examples/buttonTypes.hbs

```diff
@@ -1,4 +1,18 @@
-{{#> button btnClass="pf-is-primary"}}Primary{{/button}}
-{{#> button btnClass="pf-is-secondary"}}Secondary{{/button}}
-{{#> button btnClass="pf-is-tertiary"}}Tertiary{{/button}}
-{{#> button btnClass="pf-is-danger"}}Danger{{/button}}
\ No newline at end of file
+{{#> button btnClass="pf-is-primary"}}
+    Primary
+{{/button}}
+{{#> button btnClass="pf-is-secondary"}}
+    Secondary
+{{/button}}
+{{#> button btnClass="pf-is-tertiary"}}
+    Tertiary
+{{/button}}
+{{#> button btnClass="pf-is-danger"}}
+    Danger
+{{/button}}
+
+<span style="background-color: black; padding: 1em .5em; margin: 1em;">
```

Comment:
I think we should avoid this. If we need a dark background let's modify the workspace and not the example.

Reply:
I'd leave it white on white for now.